.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,6 @@
+{
+  "dateutil/test/test_tz.py::TzWinTest::testTzResNameFromString": true,
+  "dateutil/test/test_tz.py::TzWinTest::testTzwinName": true,
+  "dateutil/test/test_tz.py::imaginaryDateTest::testMonroviaForward": true,
+  "dateutil/test/test_utils.py": true
+}
```

Comment:
These cache files generally don’t belong in the commit.  Is this here intentionally?

Reply:
I think they are automatically generated during testing, can be deleted. I will have to tidy things up and push it again.